# Inputs
string table_name
string entry_name
string default_value
---
# Outputs
string output